collections/vessel_density_all.yaml

```diff
@@ -0,0 +1,82 @@
+Name: Vessel_Density_All
+Description: |
+  The Vessel Density maps in the EU are created since the 2019 by Cogea for the European Marine Observation and Data Network (EMODnet). The dataset is updated every year and is available for viewing and download on EMODnet Human Activities web portal (www.emodnet-humanactivities.eu). The maps are based on AIS data yearly purchased from Collecte Localisation Satellites (CLS) and ORBCOMM. The maps, GeoTIFF format, show shipping density in 1x1km cells of a grid covering all EU waters and some neighbouring areas. Density is expressed as hours per square kilometre per month. The following ship types are available:0 Other, 1 Fishing, 2 Service, 3 Dredging or underwater ops, 4 Sailing, 5 Pleasure Craft, 6 High speed craft, 7 Tug and towing, 8 Passenger, 9 Cargo, 10 Tanker, 11 Military and Law Enforcement, 12 Unknown and All ship types.
+  This particular layer refers to EMODnet ship type "All" that includes all the AIS ship Type codes.
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: vessel_density_all/README.md
+Image: vessel_density_all/vessel_density_all.png
+EDCBrowser: ""
+Resolution: 0.0083 x 0.0083°
+GeographicalCoverage: -66.949482215, 19.222315519, 72.48195026, 74.653019863
+TemporalAvailability: 2017-01-01 - 2020-12-01
+BandInformation: All
+Contact: "[Cogea srl](mailto:lfalco@cogea.it), [AUTHOR](mailto:lfalco@cogea.it)"
+Provider: "[RESOURCE PROVIDER](mailto:pdebusschere@cls.fr), [RESOURCE SOURCE](https://www.emodnet-humanactivities.eu/) "
+ManagedBy: "[ESA](http://www.esa.int/)"
+Tags:
+  - copernicus services
+  - C3S
+  - open data
+  - sentinel hub
+  - raster
+  - race challenges
+License: |
+  Re-use of content for commercial or non-commercial purposes is permitted free of charge, provided that the sources (both EMODnet - Human Activities, and CLS) are acknowledged. EMODnet - Human Activities accepts no responsibility or liability whatsoever for the re-use of content accessible on its website. ([details](https://www.emodnet-humanactivities.eu/search-results.php?dataname=Vessel+Density+))
+Resources:
+ - Group: Sentinel Hub Resources
+ - EndPoint: https://services.sentinel-hub.com
+   Name: Sentinel Hub
+   Role: processor
+   Type: byoc-beee9b47-b1ab-4044-baa5-0958d3465f12
+   CollectionId: beee9b47-b1ab-4044-baa5-0958d3465f12
+   Primary: True
+Configurations:
+  - href: "https://docs.sentinel-hub.com/api/latest/api/process/"
+    rel: "about"
+    type: "text/html"
+    title: "Details about running Evalscripts"
+  - href: ""
+    rel: "processing-expression"
+    type: "application/javascript"
+    title: "Evalscript to generate Vessel Density imagery"
+    sentinelhub:layer_name: ""

```

Comment:
This cant be empty. A link to a visualisation evalscript is needed.   Same to the rest of the YAML files

@Schpidi 
not sure whether you or Allessandro will create the script also,  

In anycase we can help create the script, but not very soon , maybe within the next week or two.

Reply:
@AlessandroScremin can you please take care of that? thanks

Reply:
both href and layer_name 

The layer name is given by you. You have done it correctly. Thanks

Reply:
both href and layer_name 

The layer name is given by you. You have done it correctly. Thanks